Content.Shared/Execution/ActiveExecutionComponent.cs

```diff
@@ -0,0 +1,19 @@
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.Execution;
+
+/// <summary>
+/// Added to a gun when it's about to fire an execution shot.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class ActiveExecutionComponent : Component
+{
+    [DataField]
+    public EntityUid Attacker;
+
+    [DataField]
+    public EntityUid Victim;
```

Comment:
why are these datafields? this component is only gonna be around for a few seconds

Reply:
As far as I know I'm supposed to make all fields that will be altered datafield even if it's not really used in stuff like prototypes although I can remove it.

Reply:
if the map is saved while executing someone it needs these to persist

Reply:
> if the map is saved while executing someone it needs these to persist

I have no idea why the map would save at the exact moment someone is being executed but I'll put them back.